x/auth/simulation/genesis.go

```diff
@@ -82,7 +82,7 @@ func RandomizedGenState(simState *module.SimulationState) {
 
 	params := types.NewParams(maxMemoChars, txSigLimit, txSizeCostPerByte,
 		sigVerifyCostED25519, sigVerifyCostSECP256K1)
-	genesisAccs := RandomGenesisAccounts(simState)
+	genesisAccs := simState.AccountGenerator(simState)
```

Comment:
this doesn't work and also is not defined. Please undo the simulation changes as my prev comment

```go
func (simState *module.SimulationState) AccountGenerator(simState *module.SimulationState)
```